docs/scarpet/api/Auxiliary.md

```diff
@@ -275,15 +276,13 @@ Available output types:
 ### `read_file(resource, type)`
 ### `delete_file(resource, type)`
 ### `write_file(resource, type, data, ...)`
-### `list_files(resource, type)`
 
 With the specified `resource` in the scripts folder, of a specific `type`, writes/appends `data` to it, reads its
- content, deletes the resource, or lists other files under this resource.
```

Comment:
why did you remove my recent changes?

Reply:
Idk, didnt do that on purpose. Ig the branch outdated, ill fix dat

Reply:
ye ill fix others at pc cos phone screen sucks for coding